### ISSUE TITLE
How do make a character sing from the left side?

### ISSUE THREAD

**Issue by lox333c** [Similirity/LITERAL-CHAOS-SOURCE-CODE#1]
## What is your suggestion, and why should it be implemented?
I want to create a character on the left side as an enemy, I write the code and writes:source/PlayState.hx:972: characters 7-13 : Unknown identifier : momDad. Similirity please help

**Comment by Similirity** [comment]
Did you make the variable for the character "mom" at the start?

I suggest you follow this video: https://www.youtube.com/watch?v=lWdvEGyAxI4
It will be kinda hard to follow since the video is in a different engine but it's simple once you learn how to do it.
